Placement tag inside deploy

I have a compose script with the following setting:
```
deploy:
            mode: global
            update_config:
                parallelism: 1
            labels:
                role: xxx
            placement:
                constraints:
                    - node.labels.monitor != xxx
```
kompose cannot parse the != properly with the following error.
```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/kubernetes/kompose/pkg/loader/compose.loadV3Placement(0xc000618010, 0x1, 0x1, 0x0)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/pkg/loader/compose/v3.go:145 +0x46a
github.com/kubernetes/kompose/pkg/loader/compose.dockerComposeToKomposeMapping(0xc0001c0b40, 0x0, 0x0, 0x0, 0x1, 0xc0004a98c0)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/pkg/loader/compose/v3.go:346 +0xc44
github.com/kubernetes/kompose/pkg/loader/compose.parseV3(0xc00004cf40, 0x1, 0x1, 0x3, 0x1, 0x0, 0x7fe1237e7000, 0xc000000300)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/pkg/loader/compose/v3.go:127 +0x1b4
github.com/kubernetes/kompose/pkg/loader/compose.(*Compose).LoadFile(0x2f72ef0, 0xc00004cf40, 0x1, 0x1, 0x0, 0x0, 0x7fe123777f3d, 0xc0002bda80, 0x2f)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/pkg/loader/compose/compose.go:186 +0x373
github.com/kubernetes/kompose/pkg/app.Up(0x100, 0x0, 0x0, 0x0, 0x0, 0x1b99785, 0x5, 0x0, 0x1bace61, 0x15, ...)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/pkg/app/app.go:270 +0x112
github.com/kubernetes/kompose/cmd.glob..func8(0x2f41840, 0xc00000d120, 0x0, 0x2)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/cmd/up.go:73 +0x4d
github.com/kubernetes/kompose/vendor/github.com/spf13/cobra.(*Command).execute(0x2f41840, 0xc00000d0e0, 0x2, 0x2, 0x2f41840, 0xc00000d0e0)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/vendor/github.com/spf13/cobra/command.go:704 +0x2d3
github.com/kubernetes/kompose/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2f41620, 0x4, 0x8, 0x0)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/vendor/github.com/spf13/cobra/command.go:785 +0x2dc
github.com/kubernetes/kompose/vendor/github.com/spf13/cobra.(*Command).Execute(0x2f41620, 0x4074a0, 0xc000092058)
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/vendor/github.com/spf13/cobra/command.go:738 +0x2b
github.com/kubernetes/kompose/cmd.Execute()
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/cmd/root.go:92 +0x2d
main.main()
        /home/wikus/seafile/files/dev/go/src/github.com/kubernetes/kompose/main.go:22 +0x20
```

--- a/pkg/loader/compose/compose_test.go
+++ b/pkg/loader/compose/compose_test.go
@@ -386,7 +386,10 @@ func TestCheckLabelsPorts(t *testing.T) {
 func TestCheckPlacementCustomLabels(t *testing.T) {
 
 	placement := types.Placement{
-		Constraints: []string{"node.labels.something == anything"},
+		Constraints: []string{
+			"node.labels.something == anything",
+			"node.labels.monitor != xxx",
+		},
 	}
 	output := loadV3Placement(placement.Constraints)
 

--- a/pkg/loader/compose/v3.go
+++ b/pkg/loader/compose/v3.go
@@ -134,8 +134,13 @@ func parseV3(files []string) (kobject.KomposeObject, error) {
 
 func loadV3Placement(constraints []string) map[string]string {
 	placement := make(map[string]string)
+	errMsg := " constraints in placement is not supported, only 'node.hostname', 'engine.labels.operatingsystem' and 'node.labels.xxx' (ex: node.labels.something == anything) is supported as a constraint "
 	for _, j := range constraints {
 		p := strings.Split(j, " == ")
+		if len(p) < 2 {
+			log.Warn(p[0], errMsg)
+			continue
+		}
 		if p[0] == "node.hostname" {
 			placement["kubernetes.io/hostname"] = p[1]
 		} else if p[0] == "engine.labels.operatingsystem" {
@@ -144,7 +149,7 @@ func loadV3Placement(constraints []string) map[string]string {
 			label := strings.TrimPrefix(p[0], "node.labels.")
 			placement[label] = p[1]
 		} else {
-			log.Warn(p[0], " constraints in placement is not supported, only 'node.hostname', 'engine.labels.operatingsystem' and 'node.labels.xxx' (ex: node.labels.something == anything) is supported as a constraint ")
+			log.Warn(p[0], errMsg)
 		}
 	}
 	return placement

